### PR TITLE
EMBCESSMOD-5151: Upgrade Suppliers Portal angular from 14 to 17

### PR DESCRIPTION
**Pull request by josephweinkam**

### CODE REVIEW

**Comment by HemanthKona** [comment]
@josephweinkam Copy the eslint.json file from registrants portal to fix the build issue.

**Comment by HemanthKona** [comment]
## Standalone Components 
Also good idea to convert all components to new standalone components, to keep it consistent with the registrants and responders portal. Please follow this guide https://angular.io/guide/standalone-migration.

**Comment by HemanthKona** [comment]
After standalone components, we need to convert the templates to use the new control flow syntax. 
```sh
ng g @angular/core:control-flow
```

File: suppliers/src/UI/embc-supplier/package.json [hunk 1]
```diff
@@ -9,46 +9,46 @@
...
+    "@angular/platform-browser": "^17.3.9",
+    "@angular/platform-browser-dynamic": "^17.3.9",
+    "@angular/router": "^17.3.9",
+    "@bcgov/bootstrap-v5-theme": "^1.0.1",
     "@fortawesome/fontawesome-free": "^5.13.0",
-    "@ng-bootstrap/ng-bootstrap": "^13.0.0",
-    "angular-oauth2-oidc": "^10.0.3",
+    "@ng-bootstrap/ng-bootstrap": "^16.0.0",
+    "angular-oauth2-oidc": "^17.0.2",
     "bootstrap": "^4.3.1",
```
Comment (HemanthKona) [inline suppliers/src/UI/embc-supplier/package.json:32]: Should we not upgrade the bootstrap package as well ?

Comment (ytqsl) [reply suppliers/src/UI/embc-supplier/package.json:32]: Created EMBCESSMOD-5432 to address this, for now this stayes with V4

**Review by ytqsl — APPROVED** [review]

**Pull request merged**